### PR TITLE
Added workflow-service to the project

### PR DESCRIPTION
**Pull request by m1yag1**
* Added a workflow service using FastAPI, w/ SQLAlchemy, uvicorn, and
Gunicorn running in Docker.
* Added docker-compose files separated out per section of docker-compose
file.
* Added .env file for each of the services.
* Added shared database postgres service.
* Added migrations folder using Alembic.
* Created initial schema for the database.
* Created ping endpoint.
* Created events endpoint with basic get.

### CODE REVIEW

File: workflow-service/LICENSE.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Sebastián Ramírez
```
Comment (mmulich) [inline workflow-service/LICENSE.txt]: Copyright Rice University?

**Review by mmulich — CHANGES_REQUESTED** [review]
There is a pyc file checked in.

I can't say I'm fully supportive of the split docker-compose config. It makes it difficult to read. Granted once you know what's happening, it's not really an issue, but the first glance understanding is not there.

I'm at the moment missing the link of how `docker-compose up -d` is finding the correct configuration file to use. Can you point that out?

File: workflow-service/LICENSE.txt (hunk 1 above)
Comment (m1yag1) [reply workflow-service/LICENSE.txt]: ah good catch. I modeled the code structure in the attribution section of the README  so licensed it as such b/c it was easier. It's not 1 for 1 copy of the cookiecutter tho. When i want to understand how things work i like to actually start from the minimum so each file is hand typed. This is my general approach to learning things ... reverse engineering =)

Comment (mmulich) [reply workflow-service/LICENSE.txt]: I'd almost leave this file out since there is a LICENSE at the root of the repo.

**Comment by m1yag1** [comment]
@pumazi I completely agree with the split out docker-compose! it was a lot for me at first and i was going to merge it all but decided to give it a change. Now, I actually really like it and see the benefits when going into the docker swarm stuff and traefik as a proxy. It takes a bit getting used to that's for sure but once you are used to it, it's nice. This is coming from someone who hates tons of files. This is a practice I'll probably keep in larger projects.

The files are discoverable by the .env file that is checked in with the repo. There is `COMPOSE_FILE` env var being set. Docker-compose supports .env files on invocation. https://docs.docker.com/compose/env-file/

<blockquote><div> Docker Documentation</div><div><strong><a href="https://docs.docker.com/compose/env-file/">Declare default environment variables in file</a></strong></div><div>Declare default environment variables in a file</div></blockquote>

**Review by mmulich — APPROVED** [review]

**Comment by mmulich** [comment]
It runs! I see interface. Good enough for me. 😸

**Pull request merged**